### ISSUE TITLE
Cannot delete user Error 500 and cannot create post with new tags

### ISSUE THREAD

**Issue by jinyeow** [fr0tt/benotes#63]
Hi,

I found a few bugs.

1. Cannot delete user, get "Failed. Error 500"
![image](https://user-images.githubusercontent.com/6980787/220479342-a321f7f6-0389-4281-a164-055e386c1d9a.png)

2. Cannot create post with new tags
I can create a post with an existing tag that's already created. But if I try to create a tag on the fly, it doesn't work.
UPDATE: just tested again as I post this, and it is now working. Haven't changed anything except that during my tests I added the tag "github", then created the post, which worked. Now tag creation in the post works - I don't know

Something else I noted was that going to the page initially shows a blank page. If I refresh the page (maybe a few times), then the login page shows. This is consistent when I test in different browsers and in private mode.

Sorry to post so much here, I am testing out Benotes and came across this during my initial setup.

Thanks
Justin

**Comment by fr0tt** [comment]
Hi,

1. I'm going to look into that
2. So is it a problem or not ? Are there particular steps that reproducible create an error ?
3. I never heard of any like that. Are you sure it has nothing to do with your setup/webserver or browser ?


 No, not at all. Thank you for pointing them out!

**Comment by jinyeow** [comment]
Hi

1. Thanks
2. No longer a problem, but may be a bug on first setup, not sure since I only set it up once.
  Steps to reproduce:
  1. Docker-compose setup
  2. After initial setup of admin account, setup the Bookmarklet
  3. Use the Bookmarklet to bookmark a website.
  4. Try to create a new tag in the Create bookmark page (NOT through the Tag page)
  The result for me was that nothing happens. If not trying to create a new tag, the post can get created.
  This issue disappeared after manually creating a new tag through the Tags page.
3. It happened across multiple browsers/sessions. I just tested and it's not happening anymore.

Sorry I don't have more details. It's not really clear why it happened on first setup but not now.
